de-lib/src/main/java/org/iplantc/de/client/models/apps/sharing/AppPermission.java

```diff
@@ -1,18 +1,24 @@
 package org.iplantc.de.client.models.apps.sharing;
 
-import com.google.web.bindery.autobean.shared.AutoBean;
+import com.google.web.bindery.autobean.shared.AutoBean.PropertyName;
 
 /**
  * Created by sriram on 2/3/16.
  */
 public interface AppPermission {
 
-    @AutoBean.PropertyName("app_id")
+    @PropertyName("app_id")
```

Comment:
Good catch! I only saw `HasQualifiedId` before and hadn't noticed that `QualifiedAppId` interface. 👍